pybliometrics/scopus/serial_title.py

```diff
@@ -129,7 +129,7 @@ def title(self):
         """The title of the source."""
         return self._entry['dc:title']
 
-    def __init__(self, issn, refresh=False, view="ENHANCED"):
+    def __init__(self, issn, years=None, refresh=False, view="ENHANCED"):
```

Comment:
The order of the parameters should be the same as in the documentation, and it's best to have `years` at the end - for consistency with the other classes